applications/asset_tracker/src/cloud_codec/cloud_codec.c

```diff
@@ -403,7 +429,10 @@ int cloud_encode_motion_data(const motion_data_t *motion_data,
 	__ASSERT_NO_MSG(motion_data != NULL);
 	__ASSERT_NO_MSG(output != NULL);
 
-	struct cloud_channel_data cloud_sensor = { .type = CLOUD_CHANNEL_FLIP };
+	struct cloud_channel_data cloud_sensor = {
+					.type = CLOUD_CHANNEL_FLIP,
+					.ts = motion_data->acceleration.ts
+					};
```

Comment:
I'd move the `}` to be at same level as `struct ...`

Reply:
Plus the indentation of members initialization is a bit awkward, one extra tab is enough.